tests/resources/primary-podify-demo.yaml

```diff
@@ -0,0 +1,27 @@
+# Save the output of this file and use kubectl create -f to import
+# it into Kubernetes.
+#
+# Created with podman-4.8.1
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: "2023-12-12T12:45:23Z"
+  labels:
+    app: podify-demo-pod
+  name: podify-demo-pod
+spec:
+  containers:
+  - env:
+    - name: HOSTNAME
+      value: d6ee99a0905d
+    - name: TERM
+      value: xterm
+    - name: HOME
+      value: /home/redis
+    image: quay.io/podman-desktop-demo/podify-demo-backend:v1
+    name: backend-podified
+    ports:
+    - containerPort: 5000
+      hostPort: 5000
```

Comment:
It works locally for you I guess, but if someone else will run it on mac, it will fail. Default behavior is that mac has 5000 port dedicated to some system application, we must take that into account. Please change host port to something else.

Reply:
switched to port 5007, hopefully that is fine